BUILDING.md

```diff
@@ -159,11 +159,11 @@ platforms. This is true regardless of entries in the table below.
 
 Depending on the host platform, the selection of toolchains may vary.
 
-| Operating System | Compiler Versions                                              |
-| ---------------- | -------------------------------------------------------------- |
-| Linux            | GCC >= 8.3                                                     |
-| Windows          | Visual Studio >= 2019 with the Windows 10 SDK on a 64-bit host |
-| macOS            | Xcode >= 11 (Apple LLVM >= 11)                                 |
+| Operating System | Compiler Versions                                           |
+| ---------------- | ----------------------------------------------------------- |
```

Comment:
I think we can keep original description since it’s a compiler bug

Reply:
sounds good, I can change it back.